packages/react-composites/tests/browser/chat/app/FakeChatAdapterArgs.ts

```diff
@@ -58,4 +58,10 @@ export type FakeChatAdapterArgs = {
    * Determines if chat composite will be using a custom data model
    */
   customDataModelEnabled?: boolean;
+  /*
+   * Record of rest errors to throw when functions of ChatThreadClient are called
+   */
+  chatThreadClientFunctionErrors?: Partial<Record<keyof ChatThreadClient, ChatThreadRestError>>;
```

Comment:
Naming nit: `chatThreadClientMethodErrors`. An interface / class has methods, not functions.